autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillManagementActivity.kt

```diff
@@ -250,7 +246,8 @@ class AutofillManagementActivity : DuckDuckGoActivity() {
 
     override fun onBackPressed() {
         when (viewModel.viewState.value.credentialMode) {
-            is Editing -> viewModel.onCancelEditMode()
+            is EditingExisting -> viewModel.onCancelEditMode()
+            is EditingNewEntry -> viewModel.onCancelManualCreation()
```

Comment:
`Editing` is now an abstract class; the concrete modes are now `EditingExisting` and `EditingNewEntry` which are both types of `Editing` modes